test/test_case.rb

```diff
@@ -38,7 +38,7 @@
   end
 end
 
-require_relative '../config/config'
+require_relative '../config/config.test'
```

Comment:
do change this line, just do in local `cp config/config.test.rb config/config.rb`